llvm/lib/Transforms/Vectorize/VPlanRecipes.cpp

```diff
@@ -900,7 +897,7 @@ Value *VPInstruction::generate(VPTransformState &State) {
     // each part of the reduction.
     unsigned UF = getNumOperands() - 1;
     VectorParts RdxParts(UF);
-    for (unsigned Part = 0; Part < UF; ++Part)
+    for (unsigned Part : seq<unsigned>(UF))
```

Comment:
Is this part of a plan to change all code in the loop vectoriser/plan to use seq? I imagine there are 100s of examples and it will start to look inconsistent if we use different styles throughout LV. @fhahn any thoughts?

I wonder if it's worth committing these changes separately?